.github/labeler.yml

```diff
@@ -0,0 +1,63 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# Pull Request Labeler Github Action Configuration: https://github.com/marketplace/actions/labeler
+build:
+  - .github/**/*
+  - .idea/**/*
+  - tools/**/*
+  - .gitignore
+  - azure-pipelines.yml
+docs:
+  - docs/**/*
+  - README.md
+base:
+  - pom.xml
+  - flink-cdc-base/**/*
+debezium:
+  - flink-cdc-debezium/**/*
+test:
+  - flink-cdc-e2e-tests/**/*
+  - flink-connector-test-util/**/*
+db2:
+  - flink-connector-db2-cdc/**/*
+  - flink-sql-connector-db2-cdc/**/*
+mongodb:
+  - flink-connector-mongodb-cdc/**/*
+  - flink-sql-connector-mongodb-cdc/**/*
+mysql:
```

Comment:
Could you refer the latest module structure? I think we should have `mysql-cdc-connector` and `mysql-pipeline-connector`

Reply:
mysql-pipeline-connector is not added yet, do we need to add label for this and other connectors in advance?

Reply:
Good catch, we can add them later